app/models/taxon_facet.rb

```diff
@@ -43,8 +43,25 @@ def query_params
     }
   end
 
+  def selected_taxon_value
+    selected_level_one_value || selected_level_two_value || selected_topic_value
```

Comment:
I think the `selected_level_two_value` needs to have precedence here, so that we show the most specific set of child taxons. Can we add a test for that?